src/lib.rs

```diff
@@ -7,7 +7,10 @@ The most simple way of using this crate is by calling the
 [`generate_simple_self_signed`] function.
 For more customization abilities, we provide the lower level
 [`Certificate::from_params`] function.
-
+*/
+#![cfg_attr(
```

Comment:
Unfortunately there seems to be no method for specifying a rustdoc example using the `///` syntax requires a feature flag enabled, so we have to use de-sugar to `cfg_attr` w/ explicit `doc`.